le_lambda.py

```diff
@@ -147,31 +132,65 @@ def lambda_handler(event, context):
                           " x_forwarded_for=\"{19}\" ssl_protocol=\"{20}\"" \
                           " ssl_cipher=\"{21}\" x_edge_response_result_type=\"{22}\"\n" \
                         .format(*line)
-                    s.sendall(log_token + msg)
+                    send_to_r7(msg, sock, LOG_TOKEN)
+                logger.info('Finished sending file={} to R7'.format(key))
             elif validate_ct_log(str(key)) is True:
+                logger.info('File={} is AWS CloudTrail log format. Parsing and sending to R7'.format(key))
                 cloud_trail = json.loads(data)
                 for event in cloud_trail['Records']:
-                    s.sendall('%s %s\n' % (log_token, json.dumps(event)))
+                    send_to_r7(json.dumps(event), sock, LOG_TOKEN)
+                logger.info('Finished sending file={} to R7'.format(key))
             else:
+                logger.info('File={} is unrecognized log format. Sending raw lines to R7'.format(key))
                 for line in lines:
-                    s.sendall('%s %s\n' % (log_token, line))
-            for token in tokens:
-                s.sendall('%s %s\n' % (token, "username='{}' finished sending log data end_time='{}'"
-                                       .format(username, str(datetime.datetime.utcnow()))))
+                    send_to_r7(line, sock, LOG_TOKEN)
+                logger.info('Finished sending file={} to R7'.format(key))
         except Exception as e:
```

Comment:
This is going to catch all the exceptions but the log message is about s3 file retrieval. I suggest either narrow the exception handling down or change the logging message to a more general one.